libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
I see what you mean.  I guess whether or not it matters overly much in this case depends on the extent to which we use those log messages to aid in the differentiation of their source.  I'm not sure.  Anyway, I like a default logger in either case. 😉